JSON_files/gear.JSON

```diff
@@ -0,0 +1,421 @@
+        "standard": -10,
+        "master_crafted": -5,
+        "artifact": 0
+      },
+      "melee_mod": {
+        "standard": 0,
+        "master_crafted": 5, // Augmented
+        "artifact": 10 // Augmented
+      },
+      "description": "An ancient set of Armorum Ferrum. Has thicker armour plating but the added weight slows down the wearer."
+    },
+    "MK4 Maximus": {
+      "armour_value": {
+        "standard": 22,
+        "master_crafted": 25,
+        "artifact": 28
+      },
+      "ranged_mod": {
+        "standard": 5,
+        "master_crafted": 10,
+        "artifact": 15
+      },
+      "melee_mod": {
+        "standard": 5,
+        "master_crafted": 10, // Augmented
+        "artifact": 15 // Augmented
+      },
+      "description": "Armour dating to the end of the Great Crusade. Often considered the ultimate Space Marine armour. The components are no longer reproducible."
+    },
+    "MK5 Heresy": {
+      "armour_value": {
+        "standard": 15,
+        "master_crafted": 17,
+        "artifact": 19
+      },
+      "ranged_mod": {
+        "standard": -5,
+        "master_crafted": 0,
+        "artifact": 0
+      },
+      "melee_mod": {
+        "standard": 20,
+        "master_crafted": 0,
+        "artifact": 0
+      },
+      "description": "A hastily assembled Power armour during the Horus Heresy to act as a stopgap. Excels at melee, alongside some downsides."
+    },
+    "MK6 Corvus": {
+      "armour_value": {
+        "standard": 15,
+        "master_crafted": 17,
+        "artifact": 19
+      },
+      "ranged_mod": {
+        "standard": 15,
+        "master_crafted": 20, // Augmented
+        "artifact": 25 // Augmented
+      },
+      "melee_mod": {
+        "standard": 0,
+        "master_crafted": 5, // Augmented
+        "artifact": 10 // Augmented
+      },
+      "description": "Relatively old beakie armour, sleek as can be. Boosted olfactory and auditory sensors increase the ranged accuracy of the wearer. Making it more fragile."
+    },
+    "MK7 Aquila": {
+      "armour_value": {
+        "standard": 17,
+        "master_crafted": 19,
+        "artifact": 21
+      },
+      "ranged_mod": {
+        "standard": 0,
+        "master_crafted": 5, // Augmented
+        "artifact": 10 // Augmented
+      },
+      "melee_mod": {
+        "standard": 0,
+        "master_crafted": 5, // Augmented
+        "artifact": 10 // Augmented
+      }
+    },
+    "MK8 Errant": {
+      "armour_value": {
+        "standard": 22,
+        "master_crafted": 24,
+        "artifact": 26
+      },
+      "ranged_mod": {
+        "standard": 0,
+        "master_crafted": 5, // Augmented
+        "artifact": 10 // Augmented
+      },
+      "melee_mod": {
+        "standard": 0,
+        "master_crafted": 5, // Augmented
+        "artifact": 10 // Augmented
+      }
+    },
+    "MK10 Indomitus": {
+      "armour_value": {
+        "standard": 24,
+        "master_crafted": 26, // Augmented
```

Comment:
these comments don't break CM during reading this file? Not all Json serializers support comments